nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
**OPTIONAL** - According to the project requirement, this method should raise the `NotImlementedError` with this text ` "#{self.class} has not implemented method '#{__method__}'"`. Please fix this 👍

Reply:
Opposite of thank you